static/src/javascripts/projects/common/modules/ui/cmp-ui.js

```diff
@@ -13,20 +12,16 @@ export const show = (forceModal: ?boolean): Promise<boolean> => {
     } else {
         require.ensure(
             [],
-            require => {
+            () => {
                 initUi = raven.context(
                     {
                         tags: {
                             feature: 'cmp',
                         },
                     },
                     () => {
-                        if (shouldUseSourcepointCmp()) {
-                            if (forceModal) {
-                                cmp.showPrivacyManager();
-                            }
-                        } else {
-                            require('common/modules/cmp-ui').init(!!forceModal);
+                        if (forceModal) {
```

Comment:
i think `forceModal` is a hangover from the old CMP – could we move the logic that sets this into here?

Reply:
in fact – does this file serve a purpose anymore? i think the idea was to lazy load the react CMP component only if it was needed but that's now gone...

@gtrufitt is that right?

Reply:
I can confirm that removing the `require.ensure` still works on DCR locally